README

```diff
@@ -32,7 +30,7 @@ To get good API documentation, run:
 
     $ epydoc -n "PyFCP API manual" -o html fcp
 
-When you install this package (refer INSTALL), you should 
+When you install this package (refer INSTALL), you should
```

Comment:
Fair enough. Will do it later though, in a more structured fashion - wanted to get this pull request in a viable state first.